src/main/java/com/gluonhq/substrate/util/ios/CodeSigning.java

```diff
@@ -109,6 +110,12 @@ public boolean signApp() throws IOException, InterruptedException {
         return sign(entitlementsPath, appPath);
     }
 
+    private void assertValidIdentity() {
+        List<Identity> identities = getIdentity();
+        if ((identities == null) || identities.isEmpty()) {
```

Comment:
remove extra parenthesis

Reply:
this makes it clear for developers who don't learn the language-specific order of evaluation by head, and it doesn't add a performance penalty, so I'd like to keep it.